tests/e2e/specs/connect-wallet-spec.cy.ts

```diff
@@ -0,0 +1,17 @@
+describe('connect wallet spec', () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('should connect wallet with success', () => {
+    cy.findByText('Arbitrum Cross-chain Message Dashboard').click();
+    cy.findByPlaceholderText('Paste your transaction hash')
+      .should('exist')
+      .type(
```

Comment:
Maybe not in this PR but eventually we will probably have to do something like [this](https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/packages/arb-token-bridge-ui/tests/support/commands.ts#L65) to bypass flaky behaviors.